app/seed-data/formula_values/data_processed/EFDB_2006_IPCC_guidelines/FormulaInputs.csv

```diff
@@ -0,0 +1,237 @@
+gas,parameter_code,parameter_name,methodology,gpc_refno,year,formula_input_value,formula_input_units,formula_name,metadata,region,actor_id,datasource,formulainput_id
```

Comment:
We don't have this table yet. Can you provide a SQL data schema for it so we can add it? Otherwise this will fail to seed I think.